package.json

```diff
@@ -92,6 +92,7 @@
     "@types/through2": "^2.0.34",
     "@types/uuid": "^3.4.4",
     "binary-search-bounds": "^2.0.4",
+    "chai": "^4.2.0",
```

Comment:
This probably belongs in `samples/package.json`

Reply:
Ah, I didn't notice that the `samples/` directory was also its own package. I took this out. It's already in `samples/package.json`, so no need to change that file.